app/models/user.rb

```diff
@@ -157,8 +157,20 @@ def full_name
     end
   end
 
+  def last_first_name
+    "#{last_name}, #{first_name}" + deactivated_string
+  end
+
   def to_s
-    full_name
+    full_name + deactivated_string
```

Comment:
There are still a lot of places where we use `full_name`. I'll probably want to clean that up at some point.